mautrix_signal/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "0.4.0"
+__version__ = "0.4.0-mod-1"
```

Comment:
I vaguely recall we used to do this in the past, but Python doesn't like non-confirming versions (it dislikes `mod` in the name)